atomicapp/providers/openshift.py

```diff
@@ -153,8 +170,7 @@ class OpenShiftProvider(Provider):
     def init(self):
         self.openshift_artifacts = {}
 
-        (self.providerapi, self.access_token, self.namespace) = \
-            self._get_config_values()
+        self._set_config_values()
```

Comment:
I was able to call the code earlier today and it passed through even though I had not set access_token in my answers. At this point we need to verify that access_token is not None.. 

If there are any others that are required to be !None then we should also verify them as well.